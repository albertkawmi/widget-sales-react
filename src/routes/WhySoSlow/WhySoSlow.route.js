import React from 'react'

const WhySoSlow = () => (
  <div>
    <h2 className="page-heading">Why so slow?</h2>
    <p>This app is hosted on Zeit's now.sh free plan. I chose this service because it makes deployment a breeze. However, they do seem to be having some <a href="https://github.com/zeit/now-cli/issues/1039">performance issues</a> of late.</p>
  </div>
)

export default WhySoSlow
