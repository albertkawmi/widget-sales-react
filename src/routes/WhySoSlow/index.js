import WhySoSLow from './WhySoSlow.route'
export default WhySoSLow
