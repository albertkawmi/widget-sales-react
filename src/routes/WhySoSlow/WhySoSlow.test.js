/* eslint-env jest */
import React from 'react'
import Enzyme from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import WhySoSlow from './WhySoSlow.route'

Enzyme.configure({ adapter: new Adapter() })

const wrapper = Enzyme.shallow(<WhySoSlow />)

describe('<WhySoSlow />', () => {
  it('renders the Why So Slow? page', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
