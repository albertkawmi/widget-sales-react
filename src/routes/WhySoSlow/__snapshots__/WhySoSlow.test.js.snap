// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WhySoSlow /> renders the Why So Slow? page 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <WhySoSlow />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h2
          className="page-heading"
>
          Why so slow?
</h2>,
        <p>
          This app is hosted on Zeit's now.sh free plan. I chose this service because it makes deployment a breeze. However, they do seem to be having some 
          <a
                    href="https://github.com/zeit/now-cli/issues/1039"
          >
                    performance issues
          </a>
           of late.
</p>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Why so slow?",
          "className": "page-heading",
        },
        "ref": null,
        "rendered": "Why so slow?",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "This app is hosted on Zeit's now.sh free plan. I chose this service because it makes deployment a breeze. However, they do seem to be having some ",
            <a
              href="https://github.com/zeit/now-cli/issues/1039"
>
              performance issues
</a>,
            " of late.",
          ],
        },
        "ref": null,
        "rendered": Array [
          "This app is hosted on Zeit's now.sh free plan. I chose this service because it makes deployment a breeze. However, they do seem to be having some ",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "performance issues",
              "href": "https://github.com/zeit/now-cli/issues/1039",
            },
            "ref": null,
            "rendered": "performance issues",
            "type": "a",
          },
          " of late.",
        ],
        "type": "p",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h2
            className="page-heading"
>
            Why so slow?
</h2>,
          <p>
            This app is hosted on Zeit's now.sh free plan. I chose this service because it makes deployment a breeze. However, they do seem to be having some 
            <a
                        href="https://github.com/zeit/now-cli/issues/1039"
            >
                        performance issues
            </a>
             of late.
</p>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Why so slow?",
            "className": "page-heading",
          },
          "ref": null,
          "rendered": "Why so slow?",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "This app is hosted on Zeit's now.sh free plan. I chose this service because it makes deployment a breeze. However, they do seem to be having some ",
              <a
                href="https://github.com/zeit/now-cli/issues/1039"
>
                performance issues
</a>,
              " of late.",
            ],
          },
          "ref": null,
          "rendered": Array [
            "This app is hosted on Zeit's now.sh free plan. I chose this service because it makes deployment a breeze. However, they do seem to be having some ",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "performance issues",
                "href": "https://github.com/zeit/now-cli/issues/1039",
              },
              "ref": null,
              "rendered": "performance issues",
              "type": "a",
            },
            " of late.",
          ],
          "type": "p",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
